sourced/ml/transformers/repo2df.py

```diff
@@ -1,33 +1,31 @@
+import operator
+
 from sourced.ml.transformers import Transformer
 
 
-class Repo2DocFreq(Transformer):
-    NDOCS_KEY = -1, 0
+class Uast2DocFreq(Transformer):
 
-    def __init__(self, extractors, **kwargs):
+    def __init__(self, extractors, document_column='blob_id', **kwargs):
         super().__init__(**kwargs)
         self.extractors = extractors
+        self.document_column = document_column
 
     def __call__(self, rows):
-        processed = rows.flatMap(self.process_row)
+        processed = rows.flatMap(self._process_row)
         if self.explained:
             self._log.info("toDebugString():\n%s", processed.toDebugString().decode())
-        reduced = processed.countByKey()
-        ndocs = None
-        for (i, key), value in reduced.items():
-            if (i, key) == self.NDOCS_KEY:
-                ndocs = value
-                continue
-            self.extractors[i].apply_docfreq(key, value)
 
-        for extractor in self.extractors:
-            extractor.ndocs = ndocs
+        df = processed \
```

Comment:
All good, just please make it a single "return" expression to be shorter (include upper line too)

Reply:
We you `processed ` in debug output (`processed.toDebugString().decode()`), so I can not join it with the upper line.

So, It will look like:
```python
    def __call__(self, rows):
        processed = rows.flatMap(self._process_row)
        if self.explained:
            self._log.info("toDebugString():\n%s", processed.toDebugString().decode())

        return processed \
            .flatMap(self._process_row) \
            .distinct() \
            .map(lambda x: (x[0], 1)) \
            .reduceByKey(operator.add)
```

Good?

Reply:
ok!